index.js

```diff
@@ -14,14 +14,32 @@
  * limitations under the License.
  */
 
-console.log('Starting server')
-const start = async () => {
-    const server = Builder()
-    try {
-      await server.listen(3000, '0.0.0.0')
-    } catch (err) {
-      server.log.error(err)
-      process.exit(1)
+'use strict'
+
+const got = require('got')
+const customService = require('@mia-platform/custom-plugin-lib')({
+  type: 'object',
+  required: ['CRUD_PATH'],
+  properties: {
+    CRUD_PATH: { type: 'string' },
+  },
+})
+
+module.exports = customService(async function index(service) {
+  service.addRawCustomPlugin('GET', '/', async(request, reply) => {
+    reply.send({ statusCode: 200, msg: 'Hello world!' })
+  })
+  service.addRawCustomPlugin('GET', '/riders', async(request, reply) => {
+    const response = await got(`${service.config.CRUD_PATH}/v2/riders/`, { json: true })
```

Comment:
We should use the http client of the custom plugin lib to make the call to the crud, not reimplement this using external dependencies

Reply:
Maybe implement the same interface as the crud (with the trailing slash) could be better